prod-aws/kafka-shared-msk/account-identity/legacy_account.tf

```diff
@@ -135,6 +135,20 @@ resource "kafka_topic" "account_identity_legacy_account_eqdb_events" {
   replication_factor = 3
 }
 
+module "cbc_account_events_relay_v2" {
+  source           = "../../../modules/tls-app"
+  consume_groups   = ["account-identity.cbc-account-events-relay-v2"]
+  consume_topics   = [kafka_topic.account_identity_legacy_account_events.name]
+  cert_common_name = "cbc/cbc-account-events-relay-v2"
+}
+
+module "cbc_fraud_detection_consumer_v1" {
+  source           = "../../../modules/tls-app"
+  consume_groups   = ["account-identity.cbc-fraud-detection-consumer-v1"]
+  consume_topics   = [kafka_topic.account_identity_public_account_events.name]
+  cert_common_name = "cbc/cbc-fraud-detection-consumer"
```

Comment:
FYI this match the cert name of an app in the cbc module: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/prod-aws/kafka-shared-msk/cbc/cbc.tf#L38

they are basically the same workload, so can't see a way of not sharing the same cert also. is that even supported?